src/utils/codeBuilder.ts

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import * as fs from 'fs';
+import { Uri } from 'vscode';
+import * as path from 'path';
+import { CompactLayoutField } from './orgUtils';
+
+type TemplateVariables = { [name: string]: string };
+
+export class CodeBuilder {
+    static readonly TEMPLATE_DIR = './resources/templates';
+    static readonly QUICK_ACTION_TEMPLATE_NAME = 'quickAction.xml';
+    static readonly LWC_DESTINATION_DIR = './force-app/main/default/lwc';
```

Comment:
I wonder if we should standardize on the familiar SFDX project path constants living in `workspaceUtils.ts`, similar to how we did for [`STATIC_RESOURCES_PATH`](https://github.com/salesforce/salesforcedx-vscode-mobile/blob/7632f668981bd3938045f0d7d3e6be6054dd1980/src/utils/workspaceUtils.ts#L13-L18)? I ran into issues where I originally started defining those paths in one of the command files, and then ended up needing them in different commands and utilities.

Reply:
I think that is a good idea. I will move these to that location.